guides/README.md

```diff
@@ -67,6 +67,17 @@ It's also possible to check links; the following command will check all links ex
 
 	make linkchecker
 
+## Building Locally Using a Container
+
+Build container image:
+
+    docker build --no-cache --progress=plain --file Containerfile --tag foreman_documentation:1.0 .
+
+Build Foreman documentation.
+Run this command in your `foreman-documentation` repository:
+
+    rm -rf guides/build && docker run --rm -v $(pwd):/foreman-documentation foreman_documentation:1.0 make html
```

Comment:
On SELinux enabled systems (at least with podman) you need to pass `-Z` for correct labels. On non-SELinux you can't pass it. Sadly that's the way it is.

Reply:
I don't know what this means & I cannot reproduce this on my machine. I've added this as a comment, but feel free to correct/reword this once more.

Reply:
I am getting this, is it related?

```
$ docker run --rm -v $(pwd):/foreman-documentation foreman_documentation:1.0 make html
make: stat: Makefile: Permission denied
make: *** No rule to make target 'html'.  Stop.
```

Option -Z is not known to my podman. Where exactly should I pass this?

Reply:
I've added separate usage instructions for SELinux enabled systems thanks to @ekohl